src/registry/registry.test.ts

```diff
@@ -0,0 +1,19 @@
+import { getFieldComponent } from './registry';
+
+jest.mock('./registry', () => ({
+  getFieldComponent: jest.fn().mockImplementation(renderType => {
+    if (renderType == 'number' || renderType == 'numeric') return 'OHRINumber';
+    if (renderType == 'checkbox' || renderType == 'multiCheckbox') return 'OHRIMultiSelect';
+  }),
+}));
+
```

Comment:
@makombe as we discussed yesterday, you don't need the stubbing above. Can you kindly update the tests so we have this merged in?